platform-db/src/main/java/com/softicar/platform/db/runtime/table/listener/DbTableRowNotificationType.java

```diff
@@ -10,16 +10,16 @@
 public enum DbTableRowNotificationType {
 
 	/**
-	 * Denotes that a {@link IDbTableRow} was deleted from the database, i.e.
-	 * <i>DELETE</i>.
+	 * Denotes that a {@link IDbTableRow} was persistently changed, that is, it
+	 * was saved with changes or loaded with changes or just deleted.
```

Comment:
```suggestion
	 * Denotes that a {@link IDbTableRow} was persistently changed. That is, it
	 * was saved with changes, or loaded with changes, or deleted.
```

Reply:
- added the commas but not the point (i think `that is` is usually appended by a comma)
- retained the `just` to express, that the delete does not require the row to be changed, as for saving and loading

Reply:
okay